### ISSUE TITLE
option to provide ocaml version as argument to `drom new`

### ISSUE THREAD

**Issue by joprice** [OCamlPro/drom#151]
Maybe there's an option I overlooked, but is there a way to provide an ocaml version to `drom new`? Otherwise, is there a command to run to update the project? I think I had to run a few commands and ultimately use the make file's `build-deps` command after deleting the local switch. Am I doing something wrong or is this something that could be improved?

**Comment by lefessan** [comment]
No, there are no such arguments to `drom new`, but I will add them: `--edition XXX` or a `--min-edition XXX` if I understand correctly.

For the update, I think you can delete the `_opam` directory, and run `drom build --edition 4.12.0` for example, it should create the switch with that version of OCaml. Is it what you want ?

**Comment by joprice** [comment]
I would think I could just update the field in the drom.toml and the rest would be handled just by running drom build. But I would understand since it is relatively rare that it could involve a manual step. Maybe that is a special case that could be detected and prompted for? 

I think the current behavior is to silently continue with the current switch, which is surprising and could be error-prone: for instance, thinking you've gotten the code to work on a new version for some time before realizing that you forgot to wipe the _opam directory.

**Comment by joprice** [comment]
Looks like a stray eprintf was left in the source when adding the min-edition flags: https://github.com/OCamlPro/drom/blob/master/src/drom_lib/commandRun.ml#L40
